ext/dom/domimplementation.c

```diff
@@ -100,8 +100,8 @@ PHP_METHOD(DOMImplementation, createDocumentType)
 	xmlFree(localname);
 
 	if (doctype == NULL) {
-		php_error_docref(NULL, E_WARNING, "Unable to create DocumentType");
-		RETURN_FALSE;
+		php_dom_throw_error(INVALID_STATE_ERR, /* strict */ true);
```

Comment:
question (you do not have to change in the same PR in case), is there a reason to keep [it as non const](https://github.com/php/php-src/blob/1b78121928ec7b3ee11d61b9e5c957c45601abd4/ext/dom/domexception.c#L46) ?

Reply:
That may be made const, and the parameter on line 33 as well.